README.md

```diff
@@ -1,24 +1,208 @@
+- FAQ
 
-* Configuration
+After you're finished please remove all the comments and instructions!
+-->
 
-* Database creation
 
-* Database initialization
 
-* How to run the test suite
+<!-- TABLE OF CONTENTS -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📗 Table of Contents
 
-* Deployment instructions
+- [📗 Table of Contents](#-table-of-contents)
+- [📖  Vespa ](#--vespa-)
+    - [Frontend Repository Link](#frontend-repository-link)
+  - [🛠 Built With ](#-built-with-)
+    - [Full-Stack Stack ](#full-stack-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* ...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  Vespa <a name="about-project"></a>
+
+> Vespa is a motorcycle rental app that allows users to add motorcycle models and cities, as well as make reservations.
+
+### Frontend Repository [Link](https://github.com/ZikaZaki/vespa-frontend)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Full-Stack Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">JS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add user has the ability to delete and add Reservation**
+- **Add user has the ability to delete and add Motorcycle model**
+- **The user has the ability to delete and add cities**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+ gem install rails
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/ZikaZaki/vespa-backend.git
+```
+-
+
+### Install
+
+Install this project with:
+
+```sh
+  cd budget-app
+  gem install
+```
+-
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  rails server
+```
+-
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec
+```
+-
```

Comment:
- [ ] Your backend app has no tests, kindly include some of them in order to ensure your app's integrity.